app/services/preserved_object_handler.rb

```diff
@@ -65,9 +65,9 @@ def create_after_validation
     elsif PreservedObject.exists?(druid: druid)
       results << result_hash(OBJECT_ALREADY_EXISTS, 'PreservedObject')
     elsif moab_validation_errors.empty?
-      results.concat(create_db_objects(Status.default_status, true))
+      results.concat(create_db_objects("ok", true))
```

Comment:
yeah, these strings here make me nervous.  Can we get to something similar to `PreservedCopy.STATUS_OK` ?

The reasoning behind the "default" status is that we're trying to be more explicit that we're not really claiming it's okay -- we're just instantiating the damn thing and don't know its status.  This particular line was in error in its original form, as we were explicitly wanting "OK".